en/report.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Translation Report
+---
+
+{% include report.html %}
```

Comment:
Sorry, but what does this do for the English language version? I think this page should be omitted

Reply:
I'm making some changes to this PR now, removing the English report is part of that 👍